### PR TITLE
source typo fixes pt2 (only on py3 merged code)

### PR DESCRIPTION
**Pull request by luzpaz**
issue 2914

### CODE REVIEW

File: src/Mod/Mesh/Gui/RemoveComponents.cpp [hunk 1]
```diff
@@ -23,13 +23,13 @@
 
 #include "PreCompiled.h"
 
-#ifndef _PreComp_
-# include <QPushButton>
+#ifndef _PreComp_
+# include <QPushButton>
 #endif
```
Comment (luzpaz) [inline src/Mod/Mesh/Gui/RemoveComponents.cpp]: Not sure why the contents of src/Mod/Mesh/Gui/RemoveComponents.cpp doesn't show the whitespace diff. Most of the correction on this file have been to whitespace with 1 typo fix interspersed.

Comment (ianrrees) [inline src/Mod/Mesh/Gui/RemoveComponents.cpp]: Because you're changing line endings.

**Comment by luzpaz** [comment]
https://github.com/looooo/FreeCAD/issues/8

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**